examples/basic_adding.py

```diff
@@ -33,7 +33,7 @@
 
 # The maximum absolute position value the strategy can accumulate in units of the coin.
 # i.e. the strategy will place orders such that it can long up to 1 ETH or short up to 1 ETH
-MAX_POSITION = 1.0
+MAX_POSITION = 0.01
```

Comment:
This seems too small and might start running into minimum order size issues. Even with 100 dollars you can do 1 ETH by just increasing leverage to 50x from default

Reply:
true, reverted